Sources/SwifterSwift/SwiftStdlib/StringExtensions.swift

```diff
@@ -957,6 +957,19 @@ public extension String {
         return range(of: pattern, options: .regularExpression, range: nil, locale: nil) != nil
     }
     #endif
+    
+    /// SwifterSwift: Overload Swift's 'contains' operator for matching regex pattern
+    ///
+    /// - Parameter lhs: String to check on regex pattern.
+    /// - Parameter rhs: Pattern to verify.
+    /// - Returns: true if string matches the pattern.
+    static func ~= (lhs: String, rhs: String) -> Bool {
```

Comment:
We already have this functionality implemented in the `matches(pattern:)` function, so feel free to copy that as it's more concise.
```Swift
return range(of: pattern, options: .regularExpression) != nil
```

You also need to wrap the function in `#if canImport(Foundation)`, since you're using `Foundation` classes/functions for it like in other functions. Don't forget to do the same for the unit tests.